src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -159,6 +160,29 @@ public async Task<Either<ActionResult, Unit>> DeleteTheme(Guid themeId)
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> DeleteUITestThemes(CancellationToken cancellationToken = default)
+        {
+            return !_themeDeletionAllowed
+                ? (Either<ActionResult, Unit>)new ForbidResult()
```

Comment:
Not asking for a change here - just spitballing - but it did take me a sec to realise where this check was taking place as I didn't expect it to be here - that could just be me. My style would be to put an if at the top to really shove it in the readers face, and remove a level of indenting from the below code - but I accept I'd then lose concision in return. I'm interested in what other devs prefer here.

I think checking `_themeDeletionAllowed` first thing makes more sense than what `DeleteTheme` does.